windows/windows_update_install/install_online_updates.yml

```diff
@@ -3,7 +3,7 @@
 ---
 - name: "Initialize Windows online updates install status"
   ansible.builtin.set_fact:
-    win_online_updates_installed: false
+    win_online_updates_succeed: false
```

Comment:
This task can be removed since this var is defined in `win_install_online_updates.yml`, and it is used after `win_install_online_updates.yml`.

Reply:
Removed, thanks.